save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt

```diff
@@ -216,5 +217,57 @@ val cardUser: FC<IndexViewProps> = FC { props ->
                 }
             }
         }
+
+        div {
+            className = ClassName("mt-2")
+            div {
+                className = ClassName("row d-flex justify-content-center text-gray-900 mt-2")
+                h5 {
+                    style = jso {
+                        textAlign = TextAlign.center
+                    }
+                    +"Your statistics:"
+                }
+            }
+            div {
+                className = ClassName("row text-muted border-bottom border-gray mx-3")
+                div {
+                    className = ClassName("col-9")
+                    p {
+                        +"Vulnerabilities: "
+                    }
+                }
+
+                div {
+                    className = ClassName("col-3")
+                    p {
+                        Link {
+                            to = "/${FrontendRoutes.PROFILE}/${props.userInfo?.name}"
```

Comment:
What if `userInfo` is null

Reply:
Fixed it

Reply:
Exactly, this is what happening on current master